project_national/fact_sheets_category_2.yml

```diff
@@ -0,0 +1,662 @@
+        lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf
+      #seal crawlspace vents
+      - &seal_vented_crawlspaces
+        option: Geometry Foundation Type|Unvented Crawlspace
+        apply_logic:
+          - Geometry Foundation Type|Vented Crawlspace
+          - or: # Ensure that the dwelling unit being modeled is either (1) not in a multifamily building, or (2) on the bottom floor of a multifamily building and therefore above the crawlspace.
+              - Geometry Building Level MF|None
+              - Geometry Building Level MF|Bottom
+        costs:
+          - value: 1.176 # $/ft2 (0.84* $/ft2 * 1.4 inflation) https://remdb.nrel.gov/measures?group_id=14&component_type_id=334&actionId=2600&bcId=5169
+            multiplier: Floor Area, Foundation (ft^2)
+        lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf
+      #insulate finished attics and cathedral ceilings to R-30
+  - upgrade_name: Advanced Enclosure with ERVs for all
+    #Up
+    options:
+      #Light Enclosure Measures
+      - *attic_insulation_IECC_CZ1A
+      - *attic_insulation_IECC_CZ2A-2B-3A-3B-3C
+      - *attic_insulation_IECC_CZ4A-4B-4C-5A-5B-6A-6B-7A-7B
+      #Intermediate Enclosure Measures
+      - *ducts_category1
+      - *ducts_category2
+      - *ducts_category3
+      - *ducts_category4
+      #- *ducts_category5
+      #- *ducts_category6
+      - *ducts_category7
+      - *ducts_category8
+      - *Drill-and-fill
+      - *foundation_wall_insulation
+      - *foundation_wall_insulation_finished_basement
+      - *rim_joist_insulation
+      - *seal_vented_crawlspaces
+      #EnergyStar Windows
+      - *EnergyStar_window_northern
+      - *EnergyStar_window_north_central
+      - *EnergyStar_window_south_central
+      - *EnergyStar_window_southern
+      - *infiltration_30pct_reduction_windows
+      - *infiltration_15pct_reduction_windows
+      
+      #1” XPS (R-5/in) Exterior wall insulation for dwelling units with wall insulation of less than R-19  
+      - &exterior_wall_insulation
+        option: Insulation Sheathing|R-5
+        apply_logic:
+          - or:
+            - Insulation Wall|Wood Stud, Uninsulated
+            - Insulation Wall|Wood Stud, R-7
+            - Insulation Wall|Wood Stud, R-11
+            - Insulation Wall|Wood Stud, R-13
+            - Insulation Wall|Wood Stud, R-15
+            - Insulation Wall|Wood Stud, Uninsulated, R-5 Sheathing
+            - Insulation Wall|Steel Stud, Uninsulated
+            - Insulation Wall|Steel Stud, R-13
+        costs:
+          - value: 0.0443 # TBD
+            multiplier: Floor Area, Attic * Insulation Increase (ft^2 * Delta R-value) # TBD
```

Comment:
This doesn't make sense. How about Wall Area, Above-Grade, Conditioned (ft^2)? The cost value seems really low also, for ref, we used $3.55 for R-30 XPS for @trynthink's ABC Simple Installation System: https://github.com/NREL/resstock/blob/23885474e2b9a76d1422c80f306cf9f03cf2257d/project_national/project_resstock_abc_sis_complete.yml#L191-L197

Reply:
I agree this doesn't look right. The value I am seeing in the costs spreadsheet is $1.85/sq ft. For the multiplier I am not sure if we want the one Lixi indicated versus "Wall Area, Above-Grade, Exterior (ft^2)" which we used in a previous project - not sure which is more appropriate, @trynthink? Would this insulation be applied to all wall area for consistency, or only to conditioned wall area?